composer/models.py

```diff
@@ -10,7 +10,7 @@
 from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
-from jmbo.managers import DefaultManager, PermittedManager
+from .managers import DefaultManager, PermittedManager
```

Comment:
Although I see other imports are already done this way, so my comments can be ignored.